packages/skyhelper/src/bot/modules/inputCommands/admin/reminders.ts

```diff
@@ -36,7 +43,16 @@ export default {
     switch (sub) {
       case "configure": {
         const ch = options.getChannel("channel", true);
-        const channel = client.channels.get(ch.id)! as APITextChannel;
+        const isThread = "parent_id" in ch;
+        const channel = client.channels.get(isThread ? ch.parent_id! : ch.id)! as APITextChannel | APIGuildForumChannel;
+
+        // Check if channel is a valid channel type
+        if (!SendableChannels.includes(channel.type)) {

```

Comment:
Needs to be this otherwise it will check for forum/parent in case of thread which will obv return false

```suggestion
        if (!SendableChannels.includes(ch.type)) {
```